docs/source/configuration.rst

```diff
@@ -33,11 +54,56 @@ Cibyl supports multiple different types of sources. Each configured in a differe
 Validate Configuration
 ^^^^^^^^^^^^^^^^^^^^^^
 
-The best way to validate the configuration you've added is correct, is to run the ``cibyl`` command. This should list the environments and systems specified in the configuration file.
+The best way to validate the configuration you've added is correct,
+is to run the ``cibyl`` command. This should list the environments
+and systems specified in the configuration file. If the configuration is
+correct, then cibyl will print the environments and systems defined in the
+configuration. Taking the minimal configuration defined in the `Format`_
+section, running cibyl will print::
```

Comment:
Change `::` to `:`.

Reply:
This is not a typo, it's rst syntax to make the next lines a literal block, so it's displayed better.

Reply:
Perfect then, I'm going to merge it +1